learn/how-tos/switch-mariadb-to-mysql-connector.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "How to Switch MariaDB to MySQL Connector"
+id: "switch-mariadb-to-mysql-connector"
+---
+---
+
+WaveMaker applications use JDBC Connectors to communicate with Databases. When you import a MySQL database, WaveMaker uses MariaDB Connector to communicate with the MySQL database. 
+
+We introduced the MariaDB JDBC connector in place of the MySQL Connector: MariaDB JDBC Connector is Apache 2.0 Licensed. 
+
+:::note
+This change is introduced in [v11.2](/learn/wavemaker-release-notes/v11-2-0).
+
+- Existing Apps will continue to use MySQL Connector
+- New apps will get Maria DB connectors, including workspace and MySQL databases.
```

Comment:
- New apps will get Maria DB connectors, for both workspace and MySQL databases.